xla/stream_executor/rocm/rocm_blas.cc

```diff
@@ -366,15 +366,18 @@ absl::Status ROCMBlas::DoBlasInternalImpl(FuncT rocblas_func, Stream *stream,
                  << ToString(ret);
     }
   }
-#if TF_ROCM_VERSION >= 60000
+#if 0
```

Comment:
+1 better to delete the code, or if you really want to keep it around, but it under if condition (that is always false), so that at least we can keep it compilable, keeping code under #if 0 bitrots too fast